.github/workflows/callable.writerside-publish.yml

```
@@ -0,0 +1,166 @@
+            git rm -rf $perv_version/
+            git commit -m "Removed old version"
+          
+            git add $version/
+            json_file="help-versions.json"
+
+            new_element=$(cat <<EOF
+          {
+          "version": "$version",
+          "url": "/\($version)\/",
+          "isCurrent": true
+          } 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)